@hubblecommerce/hubble/core/components/detail/ProductDetailBuyboxOptions.vue

```diff
@@ -58,6 +63,8 @@ export default {
             isLoading: (state) => state.modCart.isLoading,
         }),
         groups: function () {
+            if (this.dataOptions && this.dataOptions.length) return this.dataOptions;
```

Comment:
Not necessary if comment in buxbox-slot.vue is done

Reply:
Fixed in [27f554c](https://github.com/hubblecommerce/hubble-frontend-pwa/commit/27f554c1061916e8fef173ff88e0efc1db04ef0e)